tests/pragmas/tpragmas_misc.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  disabled: true
+    disabled: true
```

Comment:
Created around 2020-05 in a commit that references #14443
Changed in a commit that references #15920
Disabled around 2021-10 for CI sake

Reply:
Yeah, this is a known issue, not exactly sure it should work but pretty sure, pragmas might be still a bit under specified.

Reply:
"a bit" is certainly the word here

`let a {.foo.} = 1` should probably work, but I would have to look through this test in details to see whether expectations make sense. Ideally it could be split into stuff that works, and `knownIssue: true` file, so we continued to test parts that are currently working

Reply:
Keeping this as disabled for now.